WordPressLoginFlow/src/main/java/org/wordpress/android/login/LoginSiteAddressFragment.java

```diff
@@ -339,15 +340,7 @@ public void handleDiscoverySuccess(@NonNull String endpointAddress) {
         // hold the URL in a variable to use below otherwise it gets cleared up by endProgress
         String inputSiteAddress = mRequestedSiteAddress;
         endProgress();
-        if (mLoginListener.getLoginMode() == LoginMode.WOO_LOGIN_MODE) {
-            mLoginListener.gotConnectedSiteInfo(
-                    mConnectSiteInfoUrl,
-                    mConnectSiteInfoUrlRedirect,
-                    mConnectSiteInfoCalculatedHasJetpack
-                                               );
-        } else {
-            mLoginListener.gotXmlRpcEndpoint(inputSiteAddress, endpointAddress);
-        }
```

Comment:
Hmm, I'm missing context here on why this check is not needed anymore and we can always default to `mLoginListener.gotXmlRpcEndpoint`. Could please share some? 🙏🏼

Reply:
Sorry for not sharing context regarding this change 🙏 

This logic (`handleDiscoverySuccess`) is called only when we use the XMLRPC discovery, and we don't use it anymore for Woo (https://github.com/wordpress-mobile/WordPress-Login-Flow-Android/pull/109)

Reply:
Thanks for clarifying!